lib/browser-specific.js

```diff
@@ -383,7 +383,7 @@ function scoreBrowserSpecificFailures(runs, expectedBrowsers) {
 
   // Now do the actual walk to score the runs.
   const scores = walkTrees(runs.map(run => run.tree), '');
-  return new Map(scores.map((score, i) => [runs[i].browser_name, score]));
+  return new Map(scores.map((score, i) => [runs[i].browser_name, Math.floor(score)]));
```

Comment:
This would be a pretty significant change, and one that would require buy-in of the Interop group (with an issue on https://github.com/web-platform-tests/interop).